janus_server/src/bin/aggregation_job_driver.rs

```diff
@@ -117,6 +117,17 @@ async fn main() -> anyhow::Result<()> {
                         let (datastore, aggregation_job_driver) =
                             (Arc::clone(&datastore), Arc::clone(&aggregation_job_driver));
                         async move {
+                            if aggregation_job_lease.lease_attempts()
+                                >= config.job_driver_config.maximum_attempts_before_failure
+                            {
+                                info!(attempts = ?aggregation_job_lease.lease_attempts(),
```

Comment:
Maybe this should be `warn!` so it stands out? But then we'll be able to search for this log statement easily anyway, and also we should be emitting metrics about this so we can alert on it. So probably a `TODO` comment so we don't forget to come back and wire up a metric suffices.

Reply:
Yeah, that makes sense; we can search, but this is "exceptional" enough that it makes sense to make it stick out in the logs a bit.